### PR TITLE
Improve documentation and examples

### PR DESCRIPTION
**Pull request by yadaiio**
This pull request builds on top of: https://github.com/clue/reactphp-sqlite/pull/43 and https://github.com/clue/reactphp-tar/pull/20.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,42 +28,42 @@ $shell->execute('env | sort | head -n10')->then(function ($env) {
...
 ```bash
-$ composer install
+composer install
 ```
 
 To run the test suite, go to the project root and run:
 
 ```bash
-$ php vendor/bin/phpunit
+php vendor/bin/phpunit
```
Comment (SimonFrings) [inline README.md:66]: I think we use this version in other projects, should be apply the same here?

```suggestion
vendor/bin/phpunit
```

Comment (yadaiio) [reply README.md]: thank you for noticing this, should be fixed now.

**Review by SimonFrings — CHANGES_REQUESTED** [review]
Looks good so far, just noticed that `examples/php.php` could also need rejection handling, similar as you already did this in `examples/bash.php` & `examples/docker.php`

**Comment by yadaiio** [comment]
oops, didn't saw the `then` in there 🤗 should also has a rejection handling now, thanks @SimonFrings

**Review by SimonFrings — APPROVED** [review]
Thanks for applying the requested changes, looks good to me :+1:

**Review by clue — APPROVED** [review]
@yadaiio Thanks for looking into this, changes LGTM! :shipit:

**Pull request merged**